opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchDataType.java

```diff
@@ -34,7 +36,10 @@ public enum MappingType {
     Ip("ip", ExprCoreType.UNKNOWN),
     GeoPoint("geo_point", ExprCoreType.UNKNOWN),
     Binary("binary", ExprCoreType.UNKNOWN),
-    Date("date", ExprCoreType.TIMESTAMP),
+    Date("date", ExprCoreType.DATE),
+    Time("date", ExprCoreType.TIME),
+    Datetime("date", ExprCoreType.DATETIME),
+    Timestamp("date", ExprCoreType.TIMESTAMP),
```

Comment:
OpenSearch has no mapping types `time`, `timestamp` and `datetime`. I think this should be reverted to avoid breaking changes.

Reply:
This has been reverted accordingly.